nixos/modules/testing/test-instrumentation.nix

```
@@ -129,6 +129,9 @@ with import ../../lib/qemu-flags.nix { inherit pkgs; };
     # Make sure we use the Guest Agent from the QEMU package for testing
     # to reduce the closure size required for the tests.
     services.qemuGuest.package = pkgs.qemu_test.ga;
+
+    # Squelch warning about unset system.stateVersion
+    system.stateVersion = lib.trivial.release;
```

This breaks tests that set `system.stateVersion`.
```suggestion
    system.stateVersion = lib.mkDefault lib.trivial.release;
```
Btw, there's a `with lib` further up the file.